config/environment/index.js

```diff
@@ -4,6 +4,14 @@ const baseSettings = {
   env: process.env.NODE_ENV || 'development',
   port: process.env.PORT || 8888,
   ip: process.env.IP || '0.0.0.0',
+  secrets: {
+    session: process.env.SESSION_SECRET || 'session-secret',
+  },
+  facebook: {
```

Comment:
Let's remove this facebook section for now (until we start adding the facebook oauth stuff.